app/waf/certs/domain.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
1. We may need to create a script names \<repo home>/scripts/create_server_keypair.sh, to create it dynamically if they do not exist. 
2. The folder can be \<repo home>/data/certs/domain.[crt|key]. 
3. Also, you may need to change \<repo home>/app/waf/Dockerfile and \<repo home>/deploy/docker-compose.yml file to adept your change(mount the folder \<repo home>/data/certs/ into container). 
4. 443 is system protected port, non-privileged user may not run process to listen on it, we may need to change it > 1024. 
5. The environments `ENABLE_HTTPS` `CERT_KEY`.. should be added to \<repo home>/deploy/appcluster.rc, also should be prefixed with 'WAF_'.